resources/views/layouts/app.blade.php

```diff
@@ -14,7 +14,7 @@
 
     <!-- Styles -->
     <link href='https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css' rel='stylesheet'></link>
-    <link rel='stylesheet' type='text/css' href='/css/app.css'>
+    <link rel='stylesheet' type='text/css' href='/css/app.css'/>
```

Comment:
There should be a space before the '/'
Also, could you end the link above (on line 16) the same way <link ... /> rather than <link ...></link>